server/app/lib/utils/fusor/vm_launcher.rb

```diff
@@ -31,6 +32,8 @@ def launch_rhev_vm(params)
         set_rhev_attrs(cpu, ram, disk_size_gb)
 
         compute_attrs = create_compute_attribute(@rhev_attrs)
+        return false unless compute_attrs
```

Comment:
I recommend we return 'nil' here opposed to false, since the method is intended to return an object if it exists.

Same pattern for the other cases.